crates/fuel-core/src/service/sub_services.rs

```diff
@@ -324,6 +325,11 @@ pub fn init_sub_services(
         chain_name,
     };
 
+    const DEFAULT_GAS_PRICE_CHANGE_PERCENT: u16 = 10;
```

Comment:
I think we need to make it part of the consensus parameters. In this case we can upgrade it later across all nodes. Could you create an issue on `fuel-vm` with priority 0 please? We will do it for the next release

Reply:
And could you use one constant across `fuel-core-bin` and here please? As a default value for the `gas-price-change-percent` argument

Reply:
i think this is for the graphql api, are you sure we should be adding it to the consensus parameters?

Reply:
At some point, we will do it for the consensus parameters when we move to decentralized block production. So I think ti is okay to do that now and sync all nodes together to have the right calculations for the gas prices in the case if we decide to change it.

Reply:
okay

Reply:
https://github.com/FuelLabs/fuel-vm/issues/888

Reply:
Can we reuse the same cosntant for the `gas-price-change-percent` please?

Reply:
Do you mean remove this constant and use the value from config?

Reply:
No, use this constant here and in the argument as a default value for the argument, to link both places. And we need to put t todo with the link to the issue near the constant to be sure that it will be fixed